charts/openmetadata/templates/deployment.yaml

```diff
@@ -43,6 +43,11 @@ spec:
           {{- with .Values.extraVolumeMounts }}
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          {{- if .Values.existingSecret }}
```

Comment:
Major change here. Let's update it this way! Since open-metadata `0.8.0` release, we now support environment variables for open-metadata configs (see [here](https://github.com/open-metadata/OpenMetadata/blob/main/conf/openmetadata.yaml)).

Let's use [secrets as environment variables](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables) for  kubernetes deployments. 

For mysql, We will define 2 secrets as environment variables (MYSQL_USER and MYSQL_USER_PASSWORD).
Will add these as environment variables for container and `values.yaml` will have the below new variables -
```yaml
mysql:
  username: <value>
  password:
    secretRef: <value>
    secretKey: <value>
```
Let me know if we want to connect further!